knime_extension/src/nodes/opendata.py

```diff
@@ -686,3 +686,160 @@ def execute(self, exec_context: knext.ExecutionContext):
         gdf = get_osmnx().geocoder.geocode_to_gdf(self.placename)
         gdf = gdf.reset_index(drop=True)
         return knext.Table.from_pandas(gdf)
+
+
+############################################
+# Socrata Search
+############################################
+@knext.node(
+    name="Socrata Search",
+    node_type=knext.NodeType.SOURCE,
+    icon_path=__NODE_ICON_PATH + "Socrata Search.png",
+    category=__category,
+    after="",
+)
+@knext.output_table(
+    name="Socrata dataset list",
+    description="Socrata dataset based on search keywords",
+)
+class SocrataSearchNode:
```

Comment:
Can you please rewrite the node description to mention first what kind of data can be retrieved instead of mentioning the technology that is used first which most users won't interest.

Reply:
Socrata dataset list from  a wealth of open data resources from governments, non-profits, and NGOs around the world based on the query term.